### DISCUSSION TITLE
Throttling tile fetching

### DISCUSSION THREAD

**Discussion by eidolonFIRE** [JaffaKetchup/flutter_map_tile_caching#163] [Q&A] [answered]
My app displays many different locations at once (many FlutterMap widgets) at different LatLng locations.

Loading a page suddenly spams a tile provider for very different locations which triggers server to deny all requests. Sometimes I get blocked from tile providers for several hours.

Q: Is there some way to throttle tile fetching? Maybe limit how many concurrent requests for a tile store before they start to queue up?

**Comment by JaffaKetchup** [comment, marked as answer]
See #164 for feature request.
